src/tools/mmaps_generator/PathCommon.h

```diff
@@ -105,7 +105,7 @@ namespace MMAP
             errno = 0;
             if ((dp = readdir(dirp)) != nullptr)
             {
-                if (matchWildcardFilter(filter.c_str(), dp->d_name))
+                if (strcmp(dp->d_name, ".") != 0 && strcmp(dp->d_name, "..") != 0 && matchWildcardFilter(filter.c_str(), dp->d_name))
```

Comment:
you can use this
https://github.com/azerothcore/azerothcore-wotlk/blob/3961e0839c8106d04e6ace425c1160771b1aa558/src/common/Utilities/Util.h#L395

Reply:
These changes are made after completing the maps.